### DISCUSSION TITLE
How do i use the video feature?

### DISCUSSION THREAD

**Discussion by Jolt32** [akiver/cs-demo-manager#733] [Q&A] [answered]
how do i use the HLAE/Video part of demo manager i am lost i have no idea how to use it.

**Comment by Jolt32** [comment]
and how do i remove 2D viewer data i took 3GB i reset database gave back 1gb where is the 2gb?

**Comment by Jolt32** [comment]
i dont have much HDD space so every MB matters

**Comment by akiver** [comment]
About the video feature, I will write a more [detailed doc](https://cs-demo-manager.com/docs/guides/video.) later 

About removing positions, please read this [doc](https://cs-demo-manager.com/docs/guides/database#optimizing-the-database).

**Reply by Jolt32** [reply]
thank you :) <3

**Reply by akiver** [reply, marked as answer]
A more detailed [documentation](https://cs-demo-manager.com/docs/guides/video) is now available.

**Comment by Jolt32** [comment]
I did that delete thing but 2gb is still somewhere but i dont know where. i refreshed file explore as the data was generating thats how i know that 2gb is still laying about. sorry

**Reply by akiver** [reply]
That's because the disk space has not been "released" by Postgresql yet - it will later; you can read more about it [here](https://www.postgresql.org/docs/current/routine-vacuuming.html#VACUUM-FOR-SPACE-RECOVERY).

You can manually run the `VACUUM FULL` command to force it.

**Reply by akiver** [reply]
The next version will run `VACUUM FULL` when optimizing the database

**Reply by Jolt32** [reply]
thank you very much love the manager makes a lot easier
